### ISSUE TITLE
[java] Rule InvalidSlf4jMessageFormat false positive with concatenated message String

### ISSUE THREAD

**Issue by jeffjensen** [pmd/pmd#491]
**Rule Set:**
InvalidSlf4jMessageFormat

**Description:**
Rule InvalidSlf4jMessageFormat false positive with a concatenated message String split for formatting.  Removing the String concatenation makes it pass.

**Code Sample demonstrating the issue:**

```
final String msg = "abcd efgh a={}, b={}, c={},"
        + " d={}, e={}";
log.debug(msg, a, b, c, d, e);
```
Results in this failure message:
```
[INFO] PMD Failure: com.example:49 Rule:InvalidSlf4jMessageFormat Priority:5 Too many arguments, expected 3 arguments but have 5.
```
**Running PMD through:**
Maven

**Comment by jsotuyod** [comment]
@jeffjensen thanks for the report!

This seems to be a duplicate of #253, I'll merge the reports.
